services/core/IEEE_2030_5/README.md

```diff
@@ -0,0 +1,211 @@
+
+## Agent Installation ##
+
+The 2030.5 agent can be installed and started using an activated terminal from the root of the volttron
+git repository. Here are the steps:
+
+1. Open a terminal and navigate to the root of the volttron git repository.
+
+    ```bash
+    cd /path/to/volttron
+    ```
+
+1. Activate the virtual environment for volttron.
+
+    ```bash
+    source env/bin/activate
+    ```
+
+1. Update the config store for the 2030.5 agent.  Note we use the identity inverter1
+
+   ```bash
+   vctl config store 'inverter1' 'inverter_sample.csv' 'services/core/IEEE_2030_5/inverter_sample.csv' --csv
+   ```
+
+1. Install and start the 2030.5 agent using the `vctl` command. The `--start` option starts the agent
+   immediately after installation. The `--agent-config` option specifies the configuration file for
+   the agent. The `--vip-identity` option sets the identity of the agent on the VOLTTRON message bus.
+
+    ```bash
+    vctl install services/core/IEEE_2030_5 --start --agent-config services/core/IEEE_2030_5/example.config.yml --vip-identity inverter1
+    ```
+
+This completes the installation and startup of the 2030.5 agent.
+
+## 2030.5 Protocol ##
+
+The 2030.5 protocol uses a
+REQUEST/RESPONSE pattern meaning that all communication with the 2030.5 server will start with a
+REQUEST being sent from the client.
+
+### Communication ###
+
+The 2030.5 protocol starts with requests to the server for determining the DERProgram and DERControl
+that the client should be using.  Once the default and scheduled DERControls are known then the
+client can act according to the 2030.5 server's requirements.
+
+The following diagram illistrates the request and response from the client to the server.
+
+```mermaid
+  sequenceDiagram
+    Client->>Server: /dcap
+    Server-->Client: DeviceCapabilities
+    Client->>Server: /edev
+    Server-->Client: EndDeviceList(1) /* found */
+    Client->>Server: /edev/0
+    Server-->Client: EndDevice
+    Client->>Server: /edev/0/fsa
+    Server-->Client: FunctionSetAssignmentList
+    Client->>Server: /edev/0/fsa/0
+    Server-->Client: FunctionSetAssignment
+    Client->>Server: /edev/0/fsa/0/derp
+    Server-->Client: DERProgramList
+    Client->>Server: /derp/0
+    Server-->Client: DERProgram
+    Client->>Server: /derp/0/derc
+    Server-->Client: DERControlList
+    Client->>Server: /derp/0/dderc
+    Server-->Client: DERDefaultControl
+    Client->>Server: /derp/0/derca
+    Server-->Client: DERControlList
+```
+
+After the DERControls are found the client needs to poll the server for updates.  Depending on the
+function set, the poll rate could be different.
+
+## Agent Limitations ##
+
+This agent is not a fully compliant 2030.5 client, meaning it does not support all of the function sets
+within the 2030.5 standard.  It provides the following function sets:
+
```

Comment:
Could you also mention that this is a reference/template/initial implementation and that this was tested only with a simulated device.